### PR TITLE
chore: bumps yakcl to v0.1.2

### PR DESCRIPTION
**Pull request by tillt**

### CODE REVIEW

File: stable/kommander/requirements.lock [hunk 1]
```diff
@@ -4,13 +4,13 @@ dependencies:
   version: 0.1.10
 - name: kommander-federation
   repository: https://mesosphere.github.io/yakcl/charts
-  version: v0.1.0
+  version: 0.1.2
```
Comment (tillt) [inline stable/kommander/requirements.lock:7]: This seems unintended. Need to find out why that "v" prefix got changed.

File: stable/kommander/Chart.yaml [hunk 2]
```diff
@@ -1,10 +1,10 @@
 apiVersion: v1
-appVersion: 1.1.0
+appVersion: 1.1.2
```
Comment (makkes) [inline stable/kommander/Chart.yaml:2]:
```suggestion
appVersion: 1.1.0
```
this doesn't change.

File: stable/kommander/requirements.lock (hunk 1 above)
Comment (makkes) [reply stable/kommander/requirements.lock:7]: nvm, this file is auto-generated, anyway.

**Comment by tillt** [comment]
> you're not bumping kommander, but some kommander components…

you are perfectly right :)

**Pull request merged**